src/rpc/rawtransaction.cpp

```diff
@@ -678,7 +699,7 @@ static RPCHelpMan combinerawtransaction()
     // transaction to avoid rehashing.
     const CTransaction txConst(mergedTx);
     // Sign what we can:
-    for (unsigned int i = 0; i < mergedTx.vin.size(); i++) {
+    for (unsigned int i{0}; i < mergedTx.vin.size(); i++) {
```

Comment:
```suggestion
    for (unsigned int i{0}; i < mergedTx.vin.size(); ++i) {
```
nit: same for here `++i` should be prioritised.

Reply:
Fixed nit